### PR TITLE
production Release

### PR DESCRIPTION
**Pull request by tomahawk-pilot**
Discharge Workflow
Patient Transfer Workflow
Various other minor optimizations and changes

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## Supported Browsers
+
+| Browser | Supported          |
+| ------- | ------------------ |
+| Firefox   | :white_check_mark: |
+| Chrome   | :white_check_mark: |
+| Brave | :white_check_mark: |
+| opera | :white_check_mark: |
+| Edge | :white_check_mark: |
+| Internet Explorer | :x: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:19]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

**Comment by tomahawk-pilot** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259&bid=18870274)

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+| Safari | :x: |
+| Uc Browser | :x: |
+| Tor | :x: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:22]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x.x   | :white_check_mark: |
+| 1.0.x   | :white_check_mark: |
+
+## Supported Browsers
+
+| Browser | Supported          |
+| ------- | ------------------ |
+| Firefox   | :white_check_mark: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:14]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: src/Redux/api.tsx [hunk 4]
```diff
@@ -295,4 +299,12 @@ export default {
         path: '/api/v1/items',
         method: 'GET',
     },
+    discharge: {
+        path: '/api/v1/patient/{external_id}/discharge_summary/',
```
Comment (tomahawk-pilot) [inline src/Redux/api.tsx:303]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+| Chrome   | :white_check_mark: |
+| Brave | :white_check_mark: |
+| opera | :white_check_mark: |
+| Edge | :white_check_mark: |
+| Internet Explorer | :x: |
+| Safari | :x: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:20]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+| Uc Browser | :x: |
+| Tor | :x: |
+
+
+## Reporting a Vulnerability
+Please create a ticket at https://support.coronasafe.network
```
Comment (tomahawk-pilot) [inline SECURITY.md:26]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: src/Redux/api.tsx [hunk 7]
```diff
@@ -295,4 +299,12 @@ export default {
...
+        method: 'POST'
+    },
+    dischargePatient: {
+        path: '/api/v1/patient/{id}/discharge_patient/',
```
Comment (tomahawk-pilot) [inline src/Redux/api.tsx:307]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+| 1.0.x   | :white_check_mark: |
+
+## Supported Browsers
+
+| Browser | Supported          |
+| ------- | ------------------ |
+| Firefox   | :white_check_mark: |
+| Chrome   | :white_check_mark: |
+| Brave | :white_check_mark: |
+| opera | :white_check_mark: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:17]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+| Edge | :white_check_mark: |
+| Internet Explorer | :x: |
+| Safari | :x: |
+| Uc Browser | :x: |
+| Tor | :x: |
+
+
+## Reporting a Vulnerability
```
Comment (tomahawk-pilot) [inline SECURITY.md:25]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+## Supported Browsers
+
+| Browser | Supported          |
+| ------- | ------------------ |
+| Firefox   | :white_check_mark: |
+| Chrome   | :white_check_mark: |
+| Brave | :white_check_mark: |
+| opera | :white_check_mark: |
+| Edge | :white_check_mark: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:18]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+| ------- | ------------------ |
+| 2.x.x   | :white_check_mark: |
+| 1.0.x   | :white_check_mark: |
+
+## Supported Browsers
+
+| Browser | Supported          |
+| ------- | ------------------ |
+| Firefox   | :white_check_mark: |
+| Chrome   | :white_check_mark: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:15]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+| Brave | :white_check_mark: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:16]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+| opera | :white_check_mark: |
+| Edge | :white_check_mark: |
+| Internet Explorer | :x: |
+| Safari | :x: |
+| Uc Browser | :x: |
```
Comment (tomahawk-pilot) [inline SECURITY.md:21]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

**Comment by tomahawk-pilot** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259&bid=18870274)

File: SECURITY.md (hunk 13 above)
Comment (tomahawk-pilot) [inline SECURITY.md:21]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 11 above)
Comment (tomahawk-pilot) [inline SECURITY.md:15]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 12 above)
Comment (tomahawk-pilot) [inline SECURITY.md:16]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 10 above)
Comment (tomahawk-pilot) [inline SECURITY.md:18]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: src/Redux/api.tsx (hunk 7 above)
Comment (tomahawk-pilot) [inline src/Redux/api.tsx:307]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 1 above)
Comment (tomahawk-pilot) [inline SECURITY.md:19]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 3 above)
Comment (tomahawk-pilot) [inline SECURITY.md:14]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: src/Redux/api.tsx (hunk 4 above)
Comment (tomahawk-pilot) [inline src/Redux/api.tsx:303]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 5 above)
Comment (tomahawk-pilot) [inline SECURITY.md:20]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 9 above)
Comment (tomahawk-pilot) [inline SECURITY.md:25]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 8 above)
Comment (tomahawk-pilot) [inline SECURITY.md:17]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 2 above)
Comment (tomahawk-pilot) [inline SECURITY.md:22]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 6 above)
Comment (tomahawk-pilot) [inline SECURITY.md:26]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: src/Components/Facility/ConsultationForm.tsx [hunk 14]
```diff
@@ -306,7 +306,7 @@ export const ConsultationForm = (props: any) => {
                   />
                 </div>)}
                 <div>
-                  <InputLabel id="existing-medication-label">Medication, if any for the above-mentioned symptoms</InputLabel>
+                  <InputLabel id="existing-medication-label">History of Present Illness</InputLabel>
```
Comment (riginoommen) [inline src/Components/Facility/ConsultationForm.tsx]:
```suggestion
                  <InputLabel id="existing-medication-label">History of present illness</InputLabel>
```

File: src/Components/Facility/ConsultationForm.tsx [hunk 15]
```diff
@@ -322,7 +322,7 @@ export const ConsultationForm = (props: any) => {
                 </div>
 
                 <div>
-                  <InputLabel id="exam-details-label">Examination Details</InputLabel>
+                  <InputLabel id="exam-details-label">Examination Details and Clinical Condition</InputLabel>
```
Comment (riginoommen) [inline src/Components/Facility/ConsultationForm.tsx]:
```suggestion
                  <InputLabel id="exam-details-label">Examination details and Clinical conditions</InputLabel>
```

File: SECURITY.md (hunk 11 above)
Comment (riginoommen) [inline SECURITY.md:15]:
```suggestion
| Firefox   | :white_check_mark: |
| Chrome | :white_check_mark: |
```

File: SECURITY.md (hunk 9 above)
Comment (dauntlessnomad) [inline SECURITY.md:25]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

**Comment by dauntlessnomad** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259&bid=18870274)

File: SECURITY.md (hunk 11 above)
Comment (dauntlessnomad) [inline SECURITY.md:15]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 8 above)
Comment (dauntlessnomad) [inline SECURITY.md:17]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 6 above)
Comment (dauntlessnomad) [inline SECURITY.md:26]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: src/Redux/api.tsx (hunk 4 above)
Comment (dauntlessnomad) [inline src/Redux/api.tsx:303]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: src/Redux/api.tsx (hunk 7 above)
Comment (dauntlessnomad) [inline src/Redux/api.tsx:307]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 13 above)
Comment (dauntlessnomad) [inline SECURITY.md:21]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 12 above)
Comment (dauntlessnomad) [inline SECURITY.md:16]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 10 above)
Comment (dauntlessnomad) [inline SECURITY.md:18]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 5 above)
Comment (dauntlessnomad) [inline SECURITY.md:20]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 2 above)
Comment (dauntlessnomad) [inline SECURITY.md:22]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 3 above)
Comment (dauntlessnomad) [inline SECURITY.md:14]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

File: SECURITY.md (hunk 1 above)
Comment (dauntlessnomad) [inline SECURITY.md:19]: Codacy found an issue: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5765259)

**Pull request closed without merging**